### PR TITLE
Add the Security Policy for Security Vulnerabilities 

### PR DESCRIPTION
**Pull request by johnpeck-us-ibm**
Create SECURITY.md

   Signed-off-by: John Peck <johnpeck@us.ibm.com>

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Security Policy
+
+## Supported Versions
+
+  We support the LTS Java releases and the current Java release.
+
+## Reporting a Vulnerability
+
+To report a security vulnerability:
+  Under the repository name, click Security. If you cannot see the "Security" tab,
```
Comment (jasonkatonica) [inline SECURITY.md:10]: Line starts with an indentation that shouldnt be present. Also this line ends with a hidden space that should be removed.

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  We support the LTS Java releases and the current Java release.
+
+## Reporting a Vulnerability
+
+To report a security vulnerability:
+  Under the repository name, click Security. If you cannot see the "Security" tab, 
+select the dropdown menu, and then click Security. Click Report a Vulnerability to open the advisory form.
+Fill out the form and then Submit the report.
+
+Someone from the team will contact you ASAP about the issue.
```
Comment (jasonkatonica) [inline SECURITY.md:14]: Markdown is recommended to end with a new line after the last character.

**Review by jasonkatonica — CHANGES_REQUESTED** [review]
The DCO in the commit is prior to the commits description. The DCO is typically on the list line of the commit.

File: SECURITY.md (hunk 2 above)
Comment (johnpeck-us-ibm) [reply SECURITY.md:14]: There is a new line at the end of line 14

File: SECURITY.md (hunk 1 above)
Comment (johnpeck-us-ibm) [reply SECURITY.md:10]: Fixing

Comment (jasonkatonica) [reply SECURITY.md]: Looks good to me.

**Review by jasonkatonica — APPROVED** [review]
LGTM

**Comment by KostasTsiounis** [comment]
Will this file only be in the `main` branch? Does this mean we will be deleting it whenever we branch off of `main`?

**Comment by jasonkatonica** [comment]
> Will this file only be in the `main` branch? Does this mean we will be deleting it whenever we branch off of `main`?

The title was just edited to remove the main branch name and to make it easier to cherry pick. I expect the file to be in all our release branches.

**Review by KostasTsiounis — APPROVED** [review]
LGTM

**Pull request merged**